frontend/src/pages/NutritionSetup/__tests__/test-utils.helper.ts

```
@@ -0,0 +1,26 @@
+import type { SetupFormValues } from "../schema";
+import { describe, it, expect } from 'vitest';
+
+export const mockValues: SetupFormValues = {
+  sex: "female",
+  age: 30,
+  height_cm: 170,
+  weight_kg: 65,
+  activity: "moderate",
+  goal: "maintain",
+  diet_flags: [],
+};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)